packages/tools/src/eventListeners/segmentation/index.ts

```diff
@@ -2,10 +2,12 @@ import segmentationRepresentationModifiedEventListener from './segmentationRepre
 import segmentationDataModifiedEventListener from './segmentationDataModifiedEventListener';
 import segmentationRepresentationRemovedEventListener from './segmentationRepresentationRemovedEventListener';
 import segmentationModifiedListener from './segmentationModifiedEventListener';
+import stackImageChangeEventListener from './stackImageChangeEventListener';
```

Comment:
Can we not include stack in the naming?  We are starting to get other viewport types and making this a generic iamgeChangeEventListener means that when we have segmentations on videos and so on, we should be able to re-use the events.